src/rules/rule-no-shorthand-property-overrides/README.md

```diff
@@ -1,5 +1,7 @@
 # rule-no-shorthand-property-overrides
 
+**Deprecated: use the [`declaration-block-no-shorthand-property-overrides`](../declaration-block-no-shorthand-property-overrides/README.md) rule instead.**
+
```

Comment:
I did (kind of). I forgot `rule-trailing-semicolon` first time around, and so I patched it in master.